content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,294 @@
 ---
-title: 'Deploying to Vercel'
-metaTitle: 'How to deploy a Prisma app to Vercel'
-metaDescription: 'Learn how to deploy to Vercel a Next.js app that uses a serverless REST API in conjunction with Prisma Client.'
+title: 'Deploy to Vercel'
+metaTitle: 'Deploy to Vercel'
+metaDescription: 'Learn how to deploy a Next.js application based on Prisma Client to Vercel.'
 ---
 
 <TopBlock>
 
-In this guide, you will set up and deploy a Next.js app to [Vercel](https://vercel.com/home). The Next.js app will expose a REST API and use Prisma Client to handle fetching, creating, and deleting records from a database.
+This guide takes you through the steps to set up and deploy a serverless application that uses Prisma to [Vercel](https://vercel.com/home). The application uses Next.js to provide a serverless REST API, and Prisma Client to handle fetching, creating, and deleting records from a database.
 
-Vercel is a cloud platform for static sites and serverless functions. Vercel integrates with a GitHub repository for automatic deployments upon commits. [Next.js](https://nextjs.org/) is a full-stack React serverless framework that integrates seamlessly with Vercel.
+Vercel is a cloud platform that hosts static sites and serverless functions. Vercel integrates with a GitHub repository to allow you to deploy automatically when you make new commits. [Next.js](https://nextjs.org/) is a full-stack React-based serverless framework that integrates seamlessly with Vercel.
```

Comment:
What's the thinking here? I do see the point of referring to 'a Vercel project' rather than just 'Vercel', but I'm not sure I like 'you can configure'. It suggests some sort of extra step to me whereas the integration is actually part of the main flow in the Vercel onboarding setup.

Reply:
I've gone with:

> You can integrate a Vercel project with a GitHub repository to allow you to deploy automatically when you make new commits.

Does that work ok?

Reply:
That sounds very concrete now and something I do not question when I read the sentence. In the previous version, it felt ambiguous. I agree that configuration suggests a manual step where integration indicates it's something that's done automatically during the setup and then it just works.

So yes, your rewrite makes a lot of sense. Thanks for the feedback!